assets/js/docs-feedback.js

```diff
@@ -0,0 +1,75 @@
+/**
+ *
+ */
+$(document).ready(function(){
+    // Check that the analytics track function is available.
+    var analyticsAvailable = window.analytics && window.analytics.track &&
+                             (typeof window.analytics.track === "function");
+
+    if (analyticsAvailable) {
+        // Show the docs feedback container.
+        $("#docsFeedbackContainer").removeClass("hidden");
+
+        // For each of the button ids attach a click hanlder that shows
+        // the additional comment section.
+        ["#docsFeedbackYes", "#docsFeedbackNo"].forEach(function(key) {
+            var answer = key === "#docsFeedbackYes" ? "Yes" : "No";
+            $(key).on("click", function() {
+                $("#feedbackButtons").addClass("hidden");
+                $("#feedbackLongForm").removeClass("hidden");
+
+                // Show the additional comment section.
+                showAdditionalCommentSection(answer);
+            });
+        });
+
+        /**
+         * Submit the documentation feedback to Segment.
+         *
+         * @param {string} answer The value of the button the user clicked when giving feedback.
+         * @param {string} comments The value of the text area in the comment section. Defaults to an empty string
+         * @param {string} email The value of the email input in the comment section. Defaults to an empty string
+         */
+        function sendFeedbackToSegement(answer, comments = "", email = "") {
+            var trackingObj = {
+                answer,
+                comments,
+                email,
```

Comment:
Shorthand property names is a ES2015 feature that isn't supported in IE. Again, we probably don't care, but it's also easy enough to use the older syntax:

```
answer: answer,
comments: comments,
email: email,
```